### ISSUE TITLE
pdf-find-results-count not showing even if set to true

### ISSUE THREAD

**Issue by ppkh** [stephanrauh/ngx-extended-pdf-viewer#1507]
When setting [showFindResultsCount]="true", pdf-find-results-count is in the DOM with no width and height. The span inside the component however contains the correct search counts.

![image](https://user-images.githubusercontent.com/95472151/186390871-2d9165f3-6687-4f53-9fff-c9396aebba6b.png)

**Version info**
- 14.5.0

**Desktop (please complete the following information):**
 - Browser Edge

**Comment by ppkh** [comment]
Seems the <span> of pdf-find-results-count has the class "hidden". I could not figure out why.

**Comment by stephanrauh** [comment]
Hidden? Strange. I'm not aware of such a feature, but it sounds like responsive design. Does it have something to do with the width of your screen?

**Comment by stephanrauh** [comment]
After a quick check of my source code, I'm even more confused. Maybe the problem is your `ChangeDetectionStrategy`. Are you using `OnPush`? The relevant code snippet seems to be pretty straight-forward:

```html
<ng-template #defaultFindbarButtons>
  <pdf-find-input-area [customFindbarInputArea]="customFindbarInputArea"></pdf-find-input-area>
  <pdf-find-highlight-all [class.hidden]="!showFindHighlightAll"></pdf-find-highlight-all>
  <pdf-find-match-case [class.hidden]="!showFindMatchCase"></pdf-find-match-case>
  <pdf-find-current-page-only [class.hidden]="!showFindCurrentPageOnly"></pdf-find-current-page-only>
  <pdf-find-range [class.hidden]="!showFindPageRange"></pdf-find-range>
  <pdf-find-entire-word [class.hidden]="!showFindEntireWord"></pdf-find-entire-word>
  <pdf-find-entire-phrase [class.hidden]="!showFindEntirePhrase"></pdf-find-entire-phrase>
  <pdf-find-ignore-accents [class.hidden]="!showFindIgnoreAccents"></pdf-find-ignore-accents>
  <pdf-find-fuzzily [class.hidden]="!showFindFuzzySearch"></pdf-find-fuzzily>
  <pdf-find-results-count [class.hidden]="!showFindResultsCount"></pdf-find-results-count>
  <pdf-findbar-message-container [class.hidden]="!showFindMessages"></pdf-findbar-message-container>
</ng-template>
```

**Comment by ppkh** [comment]
Hey @stephanrauh, thanks for the quick answer. I checked our code and we indeed have OnPush on the component that uses ngx-extended-pdf-viewer. However, even with removing onPush it still doesn't work. As a workaround I found myself to modify the class manually by overriding the display value like below.

`      #findResultsCount {
        display: inherit !important;
      }`

As for the main problem I still cannot figure out why that happens.

**Comment by stephanrauh** [comment]
Strange indeed. `[class.hidden]` is a fairly standard feature of Angular, so I didn't expect any difficulties. Does triggering change detection manually help?

However, your workaround looks OK to me. The attribute `[showFindResultsCount]` is just a convenience attribute sparing you doing CSS magic. So I'm closing the issue now. If you send me a reproducer - a simple but complete Angular application demonstrating the issue - I'll reopen it and start the investigation.

**Comment by ppkh** [comment]
Alright, I could create a minimal reproduction which is on this repository: https://github.com/ppkh/pdfViewerReproduction

If you run it and check the DOM, #findResultsCount has the class "hidden". I would expect to have it shown. This repo was created with `ng new` and `npx ng add ngx-extended-pdf-viewer`.

**Comment by stephanrauh** [comment]
Thanks a lot for going the extra mile. As it turns out, this is not an Angular bug. It's a pdf.js bug (or rather, a bug of my fork of pdf.js).

**Comment by stephanrauh** [comment]
Found it! Reproducers are valuable, even such an embarrassing reproducer as this one that simply followed my "getting started" instruction without adding anything. :) The beauty of a reproducer that I don't have to look for the bug. I already know it's there. That helps a lot!

Here's the commit I've merged only partially, thereby causing the bug: https://github.com/mozilla/pdf.js/commit/6fe4b3a5ae5fb132043ba97ea8c4a068aac7993e
